README.md

```diff
@@ -146,6 +147,7 @@ See code for all available configurations.
 [Dell XPS E7240]: dell/e7240
 [Dell XPS 13 7390]: dell/xps/13-7390
 [Dell XPS 13 9343]: dell/xps/13-9343
+[Dell XPS 13 9310]: dell/xps/13-9310
```

Comment:
Can you also add this in flake.nix? Otherwise this looks good to go, if you have nothing to add.

Reply:
Done :+1: